### PR TITLE
#161348744 Setup an empty react project.

### PR DESCRIPTION
**Pull request by dnuwa**
#### What does this PR do?
sets up an empty React Project

#### Description of Task to be completed?
- setup React boilerplate
- Install and save React dependencies

#### What are the relevant pivotal tracker stories?
[#161348744](https://www.pivotaltracker.com/story/show/161348744)

### CODE REVIEW

File: .babelrc [hunk 1]
```diff
@@ -0,0 +1,6 @@
+{
+    "presets": [
+        "@babel/preset-env",
+        "@babel/preset-react"
+    ]
+}
```
Comment (judeinno) [inline .babelrc]: Add new lines at the end of the files

**Review by Sulaiman-Mozes — APPROVED** [review]

**Review by RachaelNantale — APPROVED** [review]
LGTM

**Review by brucemakallan — APPROVED** [review]
I ran your branch locally and everything is okay

**Review by judeinno — APPROVED** [review]
Thanks for working on the requested changes.

**Review by Sulaiman-Mozes — APPROVED** [review]
Nice work Daniel

File: webpack.config.js [hunk 2]
```diff
@@ -0,0 +1,39 @@
...
+    module: {
+        rules: [
+            {
+                test: /\.(js|jsx)$/,
+                exclude: /node_modules/,
+                use: {
+                    loader: "babel-loader"
+                }
+            },
+            {
```
Comment (tibetegya) [inline webpack.config.js]: You need to add an images loader

Comment (tibetegya) [reply webpack.config.js]: plus a fonts loader

File: .eslintrc [hunk 3]
```diff
@@ -0,0 +1,20 @@
+{
+    "parser": "babel-eslint",
+    "ecmaFeatures": {
```
Comment (tibetegya) [inline .eslintrc]: "ecmaFeatures" is marked as deprecated

**Review by tibetegya — COMMENTED** [review]
Es lint is flagging a number of issues, could you please address them

File: src/components/App.js [hunk 4]
```diff
@@ -0,0 +1,10 @@
+import React from 'react';
+import ReactDOM from 'react-dom';
+
+const App = () => (
+    <div>
+        <p>React here!</p>
+    </div>
+);
+export default App;
+ReactDOM.render(<App />, document.getElementById('app'));
```
Comment (tibetegya) [inline src/components/App.js]: Move the ReactDOM render method to the index.js

File: package.json [hunk 5]
```diff
@@ -0,0 +1,45 @@
...
+    "eslint": "^5.9.0",
+    "eslint-config-airbnb": "^17.1.0",
+    "eslint-plugin-import": "^2.14.0",
+    "eslint-plugin-jsx-a11y": "^6.1.2",
+    "eslint-plugin-react": "^7.11.1",
+    "html-loader": "^0.5.5",
+    "html-webpack-plugin": "^3.2.0",
+    "mini-css-extract-plugin": "^0.4.5",
+    "prettier": "^1.15.2",
+    "react": "^16.6.3",
```
Comment (tibetegya) [inline package.json]: Why is react in the dev dependencies?

File: .eslintrc (hunk 3 above)
Comment (dnuwa) [reply .eslintrc]: "ecmaFeatures" has been overridden to cater for ECMAScript 6 and enabled jsx

File: .eslintrc [hunk 6]
```diff
@@ -0,0 +1,20 @@
+{
```
Comment (tibetegya) [inline .eslintrc]: use a .eslintrc.js configuration file because the use of .eslintrc is deprecated

**Review by tibetegya — COMMENTED** [review]
@dnuwa refer to https://eslint.org/docs/user-guide/configuring for setting up eslint plus follow andela engineering best practices when writing commit message.

File: .eslintrc (hunk 6 above)
Comment (dnuwa) [reply .eslintrc]: file changed to .eslintrc.js

File: package.json (hunk 5 above)
Comment (dnuwa) [reply package.json]: react and react-dom has been moved to dependencies

File: src/components/App.js (hunk 4 above)
Comment (dnuwa) [reply src/components/App.js]: this has been moved

**Pull request merged**